src/tools/actors/ActorExecutionTrace.java

```diff
@@ -93,32 +87,23 @@ public static void recordMainActor(final Actor mainActor,
     }
   }
 
-  public static void setUpGCMonitoring() {
-    for (java.lang.management.GarbageCollectorMXBean bean : gcbeans) {
-      NotificationEmitter emitter = (NotificationEmitter) bean;
-      NotificationListener listener = new NotificationListener() {
-        @Override
-        public void handleNotification(final Notification notification, final Object handback) {
-          if (notification.getType().equals(GarbageCollectionNotificationInfo.GARBAGE_COLLECTION_NOTIFICATION)) {
-            // get the information associated with this notification
-            GarbageCollectionNotificationInfo info = GarbageCollectionNotificationInfo.from((CompositeData) notification.getUserData());
-            VM.println(Thread.currentThread().toString());
-            VM.println(info.getGcAction() + ": - " + info.getGcInfo().getId() + " " + info.getGcName() + " (from " + info.getGcCause() + ") " + info.getGcInfo().getDuration() + " ms;");
-            VM.println("GcInfo MemoryUsageBeforeGc: " + info.getGcInfo().getMemoryUsageBeforeGc().entrySet().stream().filter(ent -> !ent.getKey().equals("Compressed Class Space") && !ent.getKey().equals("Code Cache")).mapToLong(usage -> usage.getValue().getUsed()).sum() / 1024 + " kB");
-            VM.println("GcInfo MemoryUsageAfterGc: " + info.getGcInfo().getMemoryUsageAfterGc().entrySet().stream().filter(ent -> !ent.getKey().equals("Compressed Class Space") && !ent.getKey().equals("Code Cache")).mapToLong(usage -> usage.getValue().getUsed()).sum() / 1024 + " kB");
-          }
-        }
-      };
```

Comment:
What's the code delated here?
You were tracking memory usage before and after GC?
If it is easy and low overhead, I think it would be good to also have a number about how much memory was collected over time.
That would mean, we could have a number of how much memory was allocated in total, right?